rbac/management/group/relation_api_dual_write_group_handler.py

```diff
@@ -252,7 +279,10 @@ def prepare_to_delete_group(self):
             custom_ids.append(role.id)
 
         if self.group.platform_default:
-            pass  # TODO: create default bindings,
+            bootstrap_service = V2TenantBootstrapService(replicator=NoopReplicator())
+            bootstrapped_tenant = bootstrap_service.bootstrap_tenant(self.group.tenant)
+            relations_to_add = bootstrap_service.default_bindings_from_mapping(bootstrapped_tenant)
+            self.group_relations_to_add.extend(relations_to_add)
```

Comment:
I think this logic is technically correct / working, but I wonder if this should really be used via the bootstrap service. I guess I kind of touched on this in the other comment, but while that service does rely on it, it's a little awkward to make that the main source, for something relatively simple. I'm curious what it would look like if we queried for the mapping for a tenant instead here and created the relationships to remove inline.

We can again consider only adding/removing the binding tuple, which would simplify this, if needed.

Reply:
bootstrap service was decoupled from this place.
only question to what it is  binding tuple in https://github.com/RedHatInsights/insights-rbac/pull/1245#discussion_r1822379932